### ISSUE TITLE
Issues provisioning any event tokens

### ISSUE THREAD

**Issue by BrunoVicious** [dlenski/python-vipaccess#57]
I am having issues provisioning any event tokens like FT10, FT11, FT12 and UBHE. All TOTP tokens work fine though.

```
└─$ vipaccess provision -p -t FT12
Generating request...
Fetching provisioning response from Symantec server...
Getting token from response...
Decrypting token...
Checking token against Symantec server...
Traceback (most recent call last):
  File "/home/brunovicious/.local/bin/vipaccess", line 8, in <module>
    sys.exit(main())
  File "/home/brunovicious/.local/lib/python3.9/site-packages/vipaccess/__main__.py", line 238, in main
    return args.func(p, args)
  File "/home/brunovicious/.local/lib/python3.9/site-packages/vipaccess/__main__.py", line 61, in provision
    elif 'period' in otp_token and otp_token['timeskew'] > otp_token['period']/10:
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
```

**Comment by dlenski** [comment]
I assume that "event tokens" means **HOTP**?

Looks like I made a mistake in the time-skew checking in 7a979c4. Fixed in cd8a87244da5d6183f07effd0a135ec098d653ae.

**Comment by BrunoVicious** [comment]
Yes HOTP. Your fix did the trick. Thank you so much.
